stable/gke-node-termination-handler/values.yaml

```diff
@@ -24,3 +24,6 @@ resources:
   requests:
     cpu: 150m
     memory: 30Mi
+
+# systemPodGracePeriod -- A period to wait for regular pods to terminate. Recommended values: 0s-14s for preemptible VMs, 0s-(regular-vm-timeout/2) for regular VMs.
+systemPodGracePeriod: 0s
```

Comment:
I'm not sure if this is the best way to name it - maybe it should be smth like `args.systemPodGracePeriod` or so. Any suggestions?

Reply:
I think maybe just adding an `extraArgs` field and then using the `toYaml` function might be best here, rather than trying to hard-code every possible flag.